app/index.js

```diff
@@ -1018,10 +1018,12 @@ module.exports = JhipsterGenerator.extend({
             // normal CSS or SCSS?
             if (this.useSass) {
                 this.template(WEBAPP_DIR + 'scss/main.scss', WEBAPP_DIR + 'scss/main.scss');
+                this.template(WEBAPP_DIR + 'scss/bootstrap.scss', WEBAPP_DIR + 'scss/bootstrap.scss');
             }
```

Comment:
Yes thats my point this shouldn't be just for bootstrap, its should be a placeholder for any vendor files in future